cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -556,7 +563,15 @@ let interaction = Openid4vpInteraction("YourAuthRequestURIHere", config: clientC
 let query = interaction.getQuery()
 let inquirer = CredentialNewInquirer(docLoader)
 let issuedCredentials = ApiVerifiableCredentialsArray() // Would need some actual credentials for this to actually work
-let verifiablePres = inquirer.Query(query, CredentialNewCredentialsOpt(issuedCredentials))
+
+// Use this code to display the list of VCs to select which of them to send.
+let matchedRequirements = inquirer.getSubmissionRequirements(query, credentials) 
```

Comment:
so actual presentation exchange match is performed here??

Reply:
Yes it matches the given credentials against the presentation

Reply:
verifiable presentation is generating few lines below